Homework-04-Classes-Files.ipynb

```diff
@@ -21,11 +21,70 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 62,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/path/to/my/file1.txt plaintext 10\n",
+      "path/to/my/file2.txt plaintext 9\n",
+      "path/to/my/file3.txt plaintext 8\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<bound method Text.new_and_measure of <__main__.Text object at 0x11155f4a8>>"
+      ]
+     },
+     "execution_count": 62,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# your code"
+    "class Text:\n",
+    "    def __init__(self, filepath, filetype):\n",
+    "        self.path_to_file=filepath\n",
+    "        self.file_type=filetype\n",
+    "\n",
+    "    def insert_meat(self, text):\n",
+    "        self.file_text=text\n",
+    "        \n",
+    "    def file_size(self, filesize):\n",
+    "        self.file_size=filesize\n",
+    "        #this does not set the size to according to size of object at filepath becasue I do not know what that means\n",
+    "        \n",
+    "    def new_and_measure(self):\n",
+    "        tester=print(input(\"what file would you like to test against?\"))\n",
```

Comment:
`print` prints something to standard out, so you are getting input and then printing the input. Since `print` doesn't return anything, `tester` will be None and trying to do `tester.file_size` will result in an exception.